frontend/src/routes/Upload.tsx

```diff
@@ -696,6 +696,11 @@ const MetaDataEdit: React.FC<MetaDataEditProps> = ({ onSave, disabled }) => {
             <InputContainer>
                 <label htmlFor="series-field">
                     {t("series.series")}
+                    {CONFIG.upload.requireSeries && <span css={{ fontWeight: "normal" }}>
+                        {" ("}
+                        <em>{t("upload.metadata.required")}</em>
+                        {")"}
+                    </span>}
```

Comment:
This span is the same as in `TitleLabel`. Maybe extract that into a separate component inside `ui/metadata.tsx`?